javascript/javascript3/week1/hyf-homework-week1/giphy App/giphy.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="giphy.css">
+
+    <title>Document</title>
+</head>
+<body>
+    <header id="headerDesign"><h1>GIPHY APP</h1></header>
+    <form id="formCall">
+    <input id ='info'>
+    <input id ='info2'>
+    <button id = 'search'>Find</button>
+</form>
+<h2>Write Any Word to Find Picture </h2>
+    <div id ="imageInfo">
+
+    </div>
+
+    <script src="giphy.js"></script>
+</body>
+</html>
```

Comment:
Your code is fine, but try to use 'Reindent line'. So each tags can highlight in the same indent.